main.lua

```diff
@@ -15,7 +15,11 @@ local init = function()
 	}
 	for _, folder in ipairs(folders) do
 		local names = path.get_files(path.combine(PATH, folder))
-		for _, name in ipairs(names) do require(name) end
+		for _, name in ipairs(names) do
```

Comment:
Its not obvious what this code is doing. A comment would be nice. Otherwise LGTM